functest/test-data/simple_contract_with_sleep.txt

```diff
@@ -0,0 +1,47 @@
+package main
```

Comment:
why not to move all contracts to separate files?

Reply:
what? cuz code of contract crucial to the test and it's way harder to understand test when code is further away.

Reply:
Please don't do this.

Reply:
keep style of the tests intact and follow it. if you want to introduce functest/test-data/ directory then do it in a separate branch and for all tests.